internal/pkg/awsiam/download.go

```diff
@@ -0,0 +1,52 @@
+package awsiam
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	eksdv1alpha1 "github.com/aws/eks-distro-build-tooling/release/api/v1alpha1"
+	"golang.org/x/sys/unix"
+
+	"github.com/aws/eks-anywhere/internal/pkg/files"
+)
+
+const awsIamClientBinary = "aws-iam-authenticator"
+
+func DownloadAwsIamAuthClient(eksdRelease *eksdv1alpha1.Release) error {
+	k, err := getKernalName()
```

Comment:
nit: 
```suggestion
	k, err := getKernelName()
```
Same below.